src/FormBuilder/resources/js/form-builder/src/components/index.ts

```diff
@@ -1,9 +1,11 @@
-import {CurrencyControl} from '@givewp/form-builder/components/CurrencyControl';
+import {CurrencyControl} from './CurrencyControl';
+import Options from './OptionsPanel';
 
 export {default as Header} from './header';
 export {SecondarySidebar, Sidebar} from './sidebar';
 
 export default function registerComponents() {
     window.givewp.form.components = window.givewp.form.components || {};
     window.givewp.form.components.CurrencyControl = CurrencyControl;
+    window.givewp.form.components.DraggableOptionsControl = Options;
```

Comment:
The `components` property is a child `givewp` directly and not of the `form` property, so we need to remove the `.form` part of these lines because this is preventing the assets from getting compiled and throwing a few errors in the console. 


![image](https://github.com/impress-org/givewp/assets/16308864/182b4fdc-31a8-457e-a630-b67d9f1d3797)

Reply:
@glaubersilva I ended up removing the exporting of components as we're going to go about this a different way by introducing an NPM package of Form Builder components in a later PR.